docs/setup/native.md

```diff
@@ -90,6 +90,12 @@ All of the OSX instructions assume `homebrew` is your [package manager](https://
    - `brew install redis`
 1. Install Postgresql & PostGIS
    1. It is MUCH easier to use the [Postgres.app](https://postgresapp.com/downloads.html) which installs the correct combination of Postgresql and PostGIS versions.
+   	  - Navigate to: https://postgresapp.com/downloads.html
+	  - Download the Postgres.app with PostgreSQL 10, 11 and 12
+	  - OSX: After download is complete, move the Postgres.app into your applications folder 
+	  - Once moved into the applications folder, create a new server with PostgreSQL version 11 selected
```

Comment:
I'd just point them to the install docs from posgresapp - i think a developer who's going to work on the platform should be able to get the app installed.

The note on which versions to chose are important for sure.

Reply:
Updated ✔️